src/Symfony/Component/PropertyAccess/PropertyAccessor.php

```diff
@@ -788,7 +784,7 @@ private function getPropertyPath($propertyPath): PropertyPath
      *
      * @return AdapterInterface
      *
-     * @throws RuntimeException When the Cache Component isn't available
+     * @throws \RuntimeException When the Cache Component isn't available
```

Comment:
applies to 3.3 (note there's a `PropertyAccess\Exception\RuntimeException` actually).